tests/socketcall_32bit/Makefile

```diff
@@ -0,0 +1,11 @@
+TARGETS=$(patsubst %.c,%,$(wildcard *.c))
+
+LDLIBS += -lpthread
+
+all: $(TARGETS)
+
+conn: conn.c
+	$(CC) $(CFLAGS) -o $@ $^
```

Comment:
I opted for "commit early, commit often" with a baseline of the provided reproducer.  Using a specific 32-bit netcat imposes an additional dependence.